introduction/deprecated_features/README.md

```diff
@@ -23,51 +25,62 @@ in chronological order.
 
 ### Removals in CSM 1.4
 
-* [SLS](../../glossary.md#system-layout-service-sls) support for downloading and uploading credentials in the `dumpstate` and `loadstate` REST APIs
+- [SLS](../../glossary.md#system-layout-service-sls) support for downloading and uploading credentials in the `dumpstate` and `loadstate` REST APIs
 
 ### Removals in CSM 1.5
 
-* [Compute Rolling Upgrade Service (CRUS)](../../glossary.md#compute-rolling-upgrade-service-crus)
-* Deprecated [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos)
+- [Compute Rolling Upgrade Service (CRUS)](../../glossary.md#compute-rolling-upgrade-service-crus)
+- Deprecated [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos)
   v1 session template and boot set fields are no longer stored in BOS.
-  * This applied to the following deprecated BOS v1 session template fields: `cfs_branch`, `cfs_url`, `partition`
-  * This applied to the following deprecated BOS v1 boot set fields: `boot_ordinal`, `network`, `shutdown_ordinal`
-  * When upgrading to CSM 1.5, these fields were automatically removed from all BOS session
-    templates that contain them.
-  * When creating BOS v1 session templates in CSM 1.5, these fields were automatically removed.
+    - This applied to the following deprecated BOS v1 session template fields: `cfs_branch`, `cfs_url`, `partition`
+    - This applied to the following deprecated BOS v1 boot set fields: `boot_ordinal`, `network`, `shutdown_ordinal`
+    - When upgrading to CSM 1.5, these fields were automatically removed from all BOS session
+      templates that contain them.
+    - When creating BOS v1 session templates in CSM 1.5, these fields were automatically removed.
 
 ### Removals in CSM 1.6
 
-* [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos) v1
-* User Access Service
-* User Access Instance
+- [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos) v1
+- [Cray Advanced Platform Monitoring and Control (CAPMC)](../../glossary.md#cray-advanced-platform-monitoring-and-control-capmc)
+  was deprecated in CSM 1.5 and intended to be removed from CSM 1.6.  The decision was made to not
+  remove it in CSM 1.6 so that existing dependencies that have not yet transitioned to
+  [Power Control Service (PCS)](../../glossary.md#power-control-service-pcs) do not break.  Support
+  for CAPMC however, is removed starting in CSM 1.6.  We encourage everyone to transition to PCS as
```

Comment:
```suggestion
  was deprecated in CSM 1.5 and intended to be removed from CSM 1.6. The decision was made to not
  remove it in CSM 1.6 so that existing dependencies that have not yet transitioned to
  [Power Control Service (PCS)](../../glossary.md#power-control-service-pcs) do not break. Support
  for CAPMC however, is removed starting in CSM 1.6. Everyone is encouraged to transition to PCS as
```

Reply:
Fixed.

Reply:
I don't see this change -- have you pushed it up yet?